core/src/main/java/com/linecorp/armeria/internal/common/util/ChannelUtil.java

```diff
@@ -40,6 +48,8 @@
     private static final Set<ChannelOption<?>> PROHIBITED_OPTIONS;
     private static final WriteBufferWaterMark DISABLED_WRITE_BUFFER_WATERMARK =
             new WriteBufferWaterMark(0, Integer.MAX_VALUE);
+    @VisibleForTesting
+    public static final long TCP_USER_TIMEOUT_BUFFER_MILLIS = 5_000L;
```

Comment:
How about removing `public` and assign `TCP_USER_TIMEOUT_BUFFER_MILLIS` to `ChannelUtilTest.TCP_USER_TIMEOUT_BUFFER_MILLIS` so that `ClientFactoryBuilderTest` and `ServerBuildert` see the value.